src/NServiceBus.RabbitMQ.Tests/ConnectionString/ConnectionStringParserTests.cs

```diff
@@ -44,7 +44,7 @@ public void Should_fail_if_host_is_not_present()
         {
             var parser = new ConnectionStringParser(settings);
 
-            Assert.Throws<Exception>(() => parser.Parse("virtualHost=Copa;username=Copa;password=abc_xyz;port=12345;requestedHeartbeat=3"));
+            Assert.Throws<NotSupportedException>(() => parser.Parse("virtualHost=Copa;username=Copa;password=abc_xyz;port=12345;requestedHeartbeat=3"));
```

Comment:
Ah OK, so you didn't change the exception being thrown, you just changed the test to be more specific. OK, no problem, let's leave it as is. BTW - this probably should have been in a separate refactoring/tech debt commit since it's unrelated to the change.